types/shelljs/index.d.ts

```diff
@@ -835,6 +835,13 @@ export interface ExecOptions extends child.ExecOptions {
 	 */
 	silent?: boolean;
 
+	/**
+	 * Exit when command return code is non-zero
+	 *
+	 * @default false
```

Comment:
Actually, this (as well as `ailent` above) defaults to the value set in the ["global" config][1].

[1]: https://documentup.com/shelljs/shelljs#configfatal

Reply:
Hey there George (@gkalpak), 

I agree with your observation, *but*, the documentation at the shelljs website says it defaults to `false`.

I *guess* the documentation for these options at the shelljs website presumes that the global settings are not altered from their defaults.  That they are altered by the global config may well be mentioned somewhere else.

So: are you just making an observation, or are you going to *require* this be changed before it will be accepted?

Reply:
Fair enough. Let's keep it as is. If anything, the docs should be fixed as well.